components/servo/lib.rs

```diff
@@ -998,6 +999,28 @@ impl Servo {
                     webview.delegate().show_form_control(webview, form_control);
                 }
             },
+            EmbedderMsg::GetWindowRect(webview_id, response_sender) => {
+                let window_rect = if let Some(webview) = self.get_webview_handle(webview_id) {
+                    let dpi = webview.dpi();
+                    let screen = webview.delegate().screen_geometry(webview).unwrap();
```

Comment:
This `unwrap` means that by default Servo will panic as the default implementation of `fn screen_geometry` returns `None`. Please do this:

```suggestion
                    let Some(screen_geometry) = webview.delegate().screen_geometry(webview) else {
                        return Default::default();
                    }
```